index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <main id="main">
+    <h1> NGOZA & MARGARET</h1>
+    <p id="title">Microverse Trials</p>
+    <figure id="img-div">
+      <img id="image" src="img.png">

```

Comment:
Please add an alternate name for your image inside your img tag this is very important and it will help users with special needs to access your website using alternative means such as screen readers, which are devices that read the contents of a webpage aloud to users who have visual impairments.

Reply:
- [Optional] Kindly uses a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.